app/controllers/admin/api/accounts_controller.rb

```diff
@@ -225,7 +225,7 @@ def make_pending
 
     buyer_account.make_pending
 
-    respond_with(buyer_account)
+    respond_with(buyer_account.decorate)
```

Comment:
Can't you call `decorate` in the getter?
```
@buyer_account ||= buyer_accounts.find(params[:id]).decorate
```

Reply:
I am not sure that this is a good idea 😕 The decorators/representers/presenters are mostly to add stuff for the view or to respond to the requests only and I do not want to decorate at the beginning of the request and then let the rest of the code rely on something that is in the decorator...

Anyway, closing this PR because this code is very outdated at the moment. I opened different PRs and I will open more.